orchestrator-salt/src/main/resources/salt-common/salt/monitoring/systemd/cdp-prometheus.service.j2

```diff
@@ -25,7 +25,8 @@ ExecStart=/opt/cdp-prometheus/prometheus \
      --storage.agent.no-lockfile \
      --config.file=/opt/cdp-prometheus/prometheus.yml \
      --web.config.file=/opt/cdp-prometheus/prometheus-web-config.yml \
-     --web.listen-address=:{{ monitoring.agentPort }}
+     --web.listen-address=:{{ monitoring.agentPort }} \
+     >>/var/log/cdp-prometheus/cdp-prometheus.log 2>>/var/log/cdp-prometheus/cdp-prometheus.log"
```

Comment:
usually we do `2>&1` instead of redirrecting it to the same file as stdout

Reply:
fixed

Reply:
how is the log file rotated/cleaned up?

Reply:
I added the logrotate config file for cdp-prometheus, and cdp-request-signer already has log rotation built into the service.